pages/registration-summary.njk

```diff
@@ -60,6 +60,16 @@ summaryListDeclarationHelper %}
           {{ summaryListHelper(__("Main partnership contact", props.language), propsData.main_partnership_contact, "/edit/main-partnership-contact", propsData.main_partnership_contact, "main_partnership_contact", props) }}
         {% endif %}
       {% endif %}
+      {% if propsData.applicationCompletePage == null %}
+        {% set htmlValue = propsData.operator_birthdate | date(props.language, "DD MMM YYYY")if propsData.operator_birthdate else 
+          "" %}
+        {% set formattedDate = (propsData.operator_birthdate | date(props.language, "DD MMM YYYY"))if propsData.operator_birthdate %}
+        {% set regroleBirthdateUrl = "/edit/operator-name" %}
+        {% if props.cumulativeFullAnswers.registration_role == 'PARTNERSHIP' %}
+          {% set regroleBirthdateUrl = "/edit/partnership-contact-details" %}
```

Comment:
I might be missing something, but there are 3 pages where birth date can be set, the third is operator-contact-details.njk
Does it need a link back from here?

Reply:
Thats the old plan from when the the operator contact details page was only going to show the birthdate when the regrole was partnership, we opted for the new page so ill remove the references within operator-contact-details, thanks